packages/zowe-explorer/src/utils/SettingsConfig.ts

```diff
@@ -176,7 +176,40 @@ export class SettingsConfig {
         }
 
         if (workspaceIsMigrated) {
-            await SettingsConfig.setDirectValue(globals.SETTINGS_VERSION, SettingsConfig.currentVersionNumber, vscode.ConfigurationTarget.Workspace);
+            await SettingsConfig.setDirectValue(globals.SETTINGS_OLD_SETTINGS_MIGRATED, true, vscode.ConfigurationTarget.Workspace);
+        }
+    }
+
+    private static async migrateToLocalStorage(): Promise<void> {
+        ZoweLogger.trace("SettingsConfig.migrateToLocalStorage called.");
+        // Migrate persistent settings to new LocalStorage solution
+        const persistentSettings = [
+            globals.SETTINGS_DS_HISTORY,
+            globals.SETTINGS_USS_HISTORY,
+            globals.SETTINGS_JOBS_HISTORY,
+            globals.SETTINGS_COMMANDS_HISTORY,
+            globals.SETTINGS_LOGS_SETTING_PRESENTED,
```

Comment:
For each of these settings being migrated to local storage, should we add a `deprecationMessage` in package.json as documented [here](https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema)?

![image](https://github.com/zowe/vscode-extension-for-zowe/assets/22344007/88539e60-4ee8-44ba-b508-aa7c43cf00cb)

Reply:
We could add this, but the setting we are still going to be using it from the package.json for local storage, I guess I could see the markdownDeprecation being used, but not the deprecation message itself.